models/maespa/R/write.config.MAESPA.R

```diff
@@ -0,0 +1,83 @@
+#-------------------------------------------------------------------------------
+# Copyright (c) 2012 University of Illinois, NCSA.
+# All rights reserved. This program and the accompanying materials
+# are made available under the terms of the 
+# University of Illinois/NCSA Open Source License
+# which accompanies this distribution, and is available at
+# http://opensource.ncsa.illinois.edu/license.html
+#-------------------------------------------------------------------------------
+
+##-------------------------------------------------------------------------------------------------#
+##' Writes a config file for Maespa
+##'
+##' Requires a pft xml object, a list of trait values for a single model run,
+##' and the name of the file to create
+##'
+##' @name write.config.MAESPA
+##' @title Write MAESPA configuration files
+##' @param settings list of settings from pecan settings file
+##' @param run.id id of run
+##' @return configuration file for MODEL for given run
+##' @export
+##' @author Tony Gardella
+##-------------------------------------------------------------------------------------------------#
+write.config.MAESPA <- function(settings,run.id){
+  
+  #This script stands alone from workflow right now.
+  #You need to have a pecan.xml file that reads in as the settings argument
+  
+  require(PEcAn.settings)
+  # TODO
+  # add arguments that allow pecan met to be used 
+  # In template file, met is hard coded. Once met2model works, revert back to 
+  # using gsub to point to met file in database
+  # add code that will format traitvalues as necessary .dat files.
+  # 
+ 
+  
+  settings <- read.settings(settings)
+  # find out where to write run/ouput
```

Comment:
Likewise, you need to pass the settings list, not read the settings file within write.configs. If you're doing testing you should set up a separate test script that reads your default settings list and then passes it to this function, rather than change the design principles of the module.